packages/core/src/__tests__/index.test.tsx

```diff
@@ -1486,3 +1486,38 @@ it("doesn't throw if children is null", () => {
 
   expect(() => render(element).update(element)).not.toThrowError();
 });
+
+it('handles getCurrentOptions', () => {
+  const TestNavigator = (props: any): any => {
+    const { state, descriptors } = useNavigationBuilder(MockRouter, props);
+
+    return descriptors[state.routes[state.index].key].render();
+  };
+
+  const navigation = React.createRef<NavigationContainerRef>();
+
+  const second = (
+    <BaseNavigationContainer ref={navigation} onStateChange={jest.fn()}>
+      <TestNavigator>
+        <Screen name="bar" options={{ a: 'b' }}>
```

Comment:
Maybe we can add one more screen? With one screen, it will always be focused anyway